platforms/org.eclipse.mita.platform.xdk110/1.0.0/xdk110.platform

```diff
@@ -466,7 +466,6 @@ sensor MAX44009 {
  * At the moment we only support slave mode where the XDK becomes a device that can be connected to (as compared to connecting to some other device).
  */
 
-export exception NoDeviceConnectedException;
```

Comment:
This breaks the BLE generator. Please revert.

Reply:
Added exception back